phantazm-api/src/main/java/com/github/phantazmnetwork/api/config/SoundConfigProcessor.java

```diff
@@ -0,0 +1,46 @@
+package com.github.phantazmnetwork.api.config;
+
+import com.github.steanky.ethylene.core.ConfigElement;
+import com.github.steanky.ethylene.core.ConfigPrimitive;
+import com.github.steanky.ethylene.core.collection.ConfigNode;
+import com.github.steanky.ethylene.core.collection.LinkedConfigNode;
+import com.github.steanky.ethylene.core.processor.ConfigProcessException;
+import com.github.steanky.ethylene.core.processor.ConfigProcessor;
+import net.kyori.adventure.key.Key;
+import net.kyori.adventure.sound.Sound;
+import org.jetbrains.annotations.NotNull;
+
+import java.util.Objects;
+
+public class SoundConfigProcessor implements ConfigProcessor<Sound> {
+
+    private final ConfigProcessor<Key> keyConfigProcessor;
```

Comment:
also, i think it's fine to just statically reference ConfigProcessor<Key> rather than needing to construct an instance like this

Reply:
what?

Reply:
so instead of having the constructor take ConfigProcessor\<Key\> and storing it in a field, just call AdventureConfigProcessors.key() directly

Reply:
but my fine-tuned dependency injection capabilities 😭😭😭